cli/command/image/build.go

```diff
@@ -208,8 +210,12 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 			includes = append(includes, ".dockerignore", relDockerfile)
 		}
 
+		compression := archive.Uncompressed
+		if options.compress {
+			compression = archive.Gzip
```

Comment:
Just to check before we go ahead; do we expect other compression methods to be added at some point (e.g. `compress=none|gzip|bzip2`) or is that feature creep, and does not have enough advantages to make it worth thinking about?